src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentResource.kt

```diff
@@ -478,7 +478,7 @@ data class History(
   @Schema(description = "Questions asked for the questionnaire")
   val questions: List<HistoryQuestion>,
   @Schema(description = "When the questionnaire was changed")
-  val incidentChangeDate: LocalDate,
+  val incidentChangeDateTime: LocalDateTime,
   @Schema(description = "Who changed the questionnaire")
```

Comment:
Are you happy that this is a breaking change? You will break the migration service?

Reply:
I think so - I will make sure both are deployed together